k8s/helm-chart/README.md

```diff
@@ -52,57 +56,73 @@ See `../cert-manager`. Basically:
 3. define issuers (clusterwise)
 4. retrieve certs (into codepod-prod namespace)
 
-Ref: https://docs.technotim.live/posts/kube-traefik-cert-manager-le/#cert-manager
+Ref: [https://docs.technotim.live/posts/kube-traefik-cert-manager-le/#cert-manager](https://docs.technotim.live/posts/kube-traefik-cert-manager-le/#cert-manager)
 
 End result: the app-codepod-io-tls certificate.
 
 ## Install
 
 Install:
 
-    helm install codepod-staging . --namespace codepod-staging --create-namespace --values=./values.staging.yaml
+```bash
+helm install codepod-staging . --namespace codepod-staging --create-namespace --values=./values.staging.yaml
+```
 
 Upgrade:
 
-    helm upgrade codepod-staging . --namespace codepod-staging --values=./values.staging.yaml
+```bash
+helm upgrade codepod-staging . --namespace codepod-staging --values=./values.staging.yaml
+```
 
 Optionally initialize the DB (run in one api pod):
 
-    npx prisma migrate dev --name init
+```bash
+npx prisma migrate dev --name init
+```
 
 - TODO: automate this
 - TODO: restore from backup
 
 Uninstall:
 
-    helm uninstall codepod-staging
+```bash
+helm uninstall codepod-staging
+```
 
 Prod:
 
-    helm upgrade codepod-prod . --namespace codepod-prod --values=./values.prod.yaml
+```bash
+helm upgrade codepod-prod . --namespace codepod-prod --values=./values.prod.yaml
+```
 
 ## Helper scripts
 
 Alpha:
 
-    kubectl apply -f secrets.yaml -n codepod-alpha
-    helm install codepod-alpha . --namespace codepod-alpha --create-namespace --values=./values.alpha.yaml
+```bash
+kubectl apply -f secrets.yaml -n codepod-alpha
+helm install codepod-alpha . --namespace codepod-alpha --create-namespace --values=./values.alpha.yaml
 
-    helm upgrade codepod-alpha . --namespace codepod-alpha --values=./values.alpha.yaml
+helm upgrade codepod-alpha . --namespace codepod-alpha --values=./values.alpha.yaml
+```
 
 Rollback:
 
-    helm rollback -n codepod-alpha codepod-alpha
-    helm rollback -n codepod-alpha codepod-alpha 11
-    helm ls -n codepod-alpha
+```bash
+helm rollback -n codepod-alpha codepod-alpha
+helm rollback -n codepod-alpha codepod-alpha 11
+helm ls -n codepod-alpha
+```
 
 To access prisma:
 
+```bash
     kubectl port-forward prisma-deployment-5c9ccfc6b8-962vq 5555:5555 -n codepod-alpha
```

Comment:
You forgot to remove this indentation.

Also, I deliberately used the `indentation` instead of ` ``` ` for small code blocks like these `helm` ones, because:
- indentation is easier and cleaner to read in plain text then ` ``` `
- ` ```bash` does not give it syntax highlighting anyway

Reply:
Both work. You could remove this `indentation` or change the small & unhighlighted commands back to `indentation`.

Reply:
Sorry for the inconvenience!

Eventually I opted to the first option which is to remove the indentation.

Reply:
No problem, thanks!